### ISSUE TITLE
Improve readme

### ISSUE THREAD

**Issue by abhi18av** [nextflow-hub/server_setup#1]
@emilyncosta , your and @Ankita-1211 suggestions would be great regarding the `Readme` / `usage documentation`.  Do you find it easy to understand and use?

Your needs would help @sharma-shruti  and @abhi18av  to think about things like

-  would you prefer GIF for this? 

- or do you need a video for server setup in https://github.com/nextflow-hub/minimal-nextflow-setup ?

**Comment by Ankita-1211** [comment]
The Readme seems fine to me, may be cuz of my previous familiarity with all this.
Video will be more user friendly, but one thing that comes to my mind  while setting up , you got to pause & play which sometimes bothers.

Each Step with gif will work  best i guess. Can include  video link too. Serving everything for varied user. (dont  know about feasibility & work it will take to do both :)

What say team @emilyncosta @sharma-shruti ???

**Comment by abhi18av** [comment]
@Ankita-1211 , yeah me and @sharma-shruti  are thinking about it - will probably add the changes by Sunday.
